tests/lib/__init__.py

```diff
@@ -353,6 +353,26 @@ def assert_installed(self, pkg_name, editable=True, with_files=[],
                     .format(**locals())
                 )
 
+    def did_create(self, path, message=None):
+        return '{}\n{}'.format(message, self) if message else self
```

Comment:
This line is being repeated across all functions, so it's better to pull this out in its own function, and then call the function here.